2_observations/src/munge_flow_dat.R

```diff
@@ -0,0 +1,19 @@
+# munge flow dat
+get_flow_sites <- function(flow_ind, temp_sites_ind, out_ind) {
```

Comment:
feels weird that this is in the `munge_flow_dat.R` file, but the `munge_flow` function is not. Maybe we should move the function here or rename this file or move this function in the `data_munge_functions.R` file. Since that file isn't that big, and since you don't have a file for temp (like `munge_temp_dat.R`) I think it would make sense to move this fxn into the `datat_munge_functions.R` file.